tests/ui/invalid_generic.stderr

```diff
@@ -0,0 +1,5 @@
+error: expected type to bind to, expected one of: `for`, parentheses, `fn`, `unsafe`, `extern`, identifier, `::`, `<`, square brackets, `*`, `&`, `!`, `impl`, `_`, lifetime
+ --> tests/ui/invalid_generic.rs:1:37
+  |
+1 | #[derive_where::derive_where(Clone; where)]
+  |                                     ^^^^^
```

Comment:
This looks a bit strange, we would only expect a type here, not parentheses ??

Reply:
I'm not sure what you mean, I don't see any parentheses. I used `where` here as something that can't be parsed into a type.

Reply:
I meant the `expected one of: "..."` but that is probably just the parser returning that. I think we can keep it like that as the error message pointing at the `where` and requiring a Type being pretty clear.

Reply:
Ah, I see. Good question, I have no clue, but yeah, this error message isn't in my control.